lib/controller.rb

```diff
@@ -83,15 +83,16 @@ class SlowFood < Sinatra::Base
     redirect '/auth/login'
   end
 
+  get '/menu' do
+    @menu = Menu.first
+    @dishes = Dish.all
```

Comment:
Oh I see, you made two routes.... So what I meant is that instead of using the `Dish.all` database call, you should use `@menu.dishes`. Then in the view, loop through `@menu.dishes` instead of `@dishes`.